src/cosmicqc/frame.py

```diff
@@ -486,7 +486,7 @@ def create_figure_group_html(
         # return the path of the file
         return report_path
 
-    def create_threshold_set_outlier_visualization(  # noqa: PLR0913
+    def create_threshold_set_outlier_visualization(
```

Comment:
Why was this removed?

Reply:
This was an update which seemed to happen as a result of updating the `ruff` pre-commit configuration. It was originally added to avoid a failure with a `ruff` check, and since `ruff` removed it without an error I trust it was the correct update to make.